server/views/pages/analytics/changePrisonGroup.njk

```diff
@@ -0,0 +1,31 @@
+{% extends "../../partials/analytics/layout.njk" %}
```

Comment:
i think you want `../../partials/layout.njk` cos doesn't the analytics one bring in the levels/entries/PC tabs?

Reply:
Good spot